apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.UnitTests/TestLogger.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using Google.Cloud.Spanner.V1.Logging;
+using Xunit.Abstractions;
+
+namespace Google.Cloud.Spanner.Data.Tests
+{
+    internal class TestLogger : DefaultLogger
+    {
+        private static Lazy<TestLogger> s_Instance = new Lazy<TestLogger>(
+            () => new TestLogger());
```

Comment:
this is a copy of the one in integrationtests since I dont want to manage them referencing each other (yet) for just this class.  static is fine.